### PR TITLE
Gelişme

### PR DESCRIPTION
**Pull request by SMSTUNITS**
Güvenlik Güncelemeleri

### CODE REVIEW

**Comment by ErcetinGuler** [comment]
neden kapattın

**Comment by aibaars** [comment]
> neden kapattın

I closed it because the pull request looks like automatically generated spam. Apologies if it isn't. I see you have made contributions to other repositories. Could you please follow the advice in https://github.com/github/vscode-codeql/pull/1185#issuecomment-1059424858 .

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# For most projects, this workflow file will not need changing; you simply need
```
Comment (aibaars) [inline .github/workflows/codeql.yml:1]: Please delete this file, we already run CodeQL, see https://github.com/github/codeql/blob/ef227a47219aef17c62bfe881b6459ef20ceb828/.github/workflows/codeql-analysis.yml .

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Security Policy
```
Comment (aibaars) [inline SECURITY.md:1]: Please remove this file from the pull request.

File: ruby/extractor/Cargo.toml [hunk 3]
```diff
@@ -9,12 +9,12 @@ edition = "2018"
 [dependencies]
 flate2 = "1.0"
 node-types = { path = "../node-types" }
-tree-sitter = "0.19"
+tree-sitter = "0.20"
 tree-sitter-embedded-template = "0.19"
 tree-sitter-ruby = { git = "https://github.com/tree-sitter/tree-sitter-ruby.git", rev = "bb6a42e42b048627a74a127d3e0184c1eef01de9" }
-clap = "2.33"
+clap = "3.0"
```
Comment (aibaars) [inline ruby/extractor/Cargo.toml:15]: The two `Cargo.toml` files have conflicts, could you resolve these please. In addition, the [Cargo.lock](https://github.com/github/codeql/blob/ef227a47219aef17c62bfe881b6459ef20ceb828/ruby/Cargo.lock) file needs to be updated to reflect the dependency changes.

**Review by aibaars — CHANGES_REQUESTED** [review]
Could you squash the commit history of your pull request and address the comments?

**Comment by ErcetinGuler** [comment]
umarım olmuştur

**Pull request closed without merging**